data_platform/ingestion/data_dumps/reddit/process_dump.py

```
@@ -0,0 +1,156 @@
+    Raises
+    ------
+    FileNotFoundError
+        When ``input_path`` is not a file.
+    FileExistsError
+        When ``output_path`` already exists.
+    """
+    _require_processable_paths(input_path, output_path)
+    sampled = reservoir_sample(
+        _iter_kept_dump_comments(input_path),
+        sample_size,
+        Random(sample_seed),
```

Ruff (0.16.3)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)